### ISSUE TITLE
Configurable root volume size for OpenStack

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/microservices-infrastructure#790]
It should be possible to specify the size of root volumes on openstack clusters by specifying `control_volume_size` and `worker_volume_size` in the Terraform template. See #723

**Comment by stevendborrelli** [comment]
Isn't this fixed by #805?

**Comment by ryane** [comment]
#805 lets you configure the size of the lvm volumes but the size of the boot volume is not yet configurable. it may not be as important now that we have the lvm support

**Comment by corebug** [comment]
@ryane well, basically in CiscoCloud's OpenStack we've got instance's root volume size predefined within instance's flavor and i don't know of any ways to customize this in deployment process.

**Comment by ryane** [comment]
I don't know openstack very well - is this a limitation of the CiscoCloud deployment or would this apply to every OpenStack deployment? It does look like the terraform provider supports a `volume_size` for the boot volume: https://www.terraform.io/docs/providers/openstack/r/compute_instance_v2.html#block_device

**Comment by BrianHicks** [comment]
It would make sense to me that this is taken care of by the external volumes. We're going to be moving more of the storage-heavy components to that. Do we want to keep this open until, say, Mesos and Docker are on LVM?

**Comment by ryane** [comment]
docker is on lvm; we are waiting on mesos in #820.

**Comment by ryane** [comment]
I agree. While it would be nice to have consistent support across the cloud providers, if this is not a relatively simple tweak to the terraform config, then we can probably close this after we get mesos over to lvm

**Comment by siddharthist** [comment]
This has been implemented: https://github.com/CiscoCloud/microservices-infrastructure/blob/e53b7da545c1bdc71a5ceff7278ace5705117b41/terraform/openstack-modules.sample.tf#L64
